ledger/ledger-on-sql/BUILD.bazel

```diff
@@ -269,7 +246,12 @@ da_scala_test_suite(
             ],
         ),
         conformance_test(
-            name = "conformance-test-participant-pruning-{}".format(db["name"]),
+            name = "conformance-test-{}".format(db["name"]),
+            lf_versions = [
```

Comment:
I am assuming that beefing up this test with both more lf versions and test cases was intentional?

Reply:
This allows unifying pruning-only suite and the conformance test one, that included multiple LF versions.